packages/web/docs/src/pages/docs/self-hosting/changelog.mdx

```diff
@@ -0,0 +1,35 @@
+import { readFileSync } from 'node:fs';
+import { buildDynamicMDX } from 'nextra/remote';
+import { RemoteContent } from '@theguild/components';
+
+export async function getStaticProps () {
+  const data = readFileSync(
+    __dirname + '/../../../../../../../../deployment/CHANGELOG.md',
```

Comment:
`__dirname` is not what you expect (`packages/web/docs/src/pages/docs/self-hosting`), but `packages/web/docs/.next/server/pages/docs/self-hosting`.

Reply:
`process.cwd(), '..', 'deployment', 'CHANGELOG.md'` maybe more simpler?